rest_api/test/samples/pipeline/test_pipeline.yaml

```diff
@@ -0,0 +1,103 @@
+version: '0.7'
+
+components:
+  - name: Reader
+    type: FARMReader
+    params:
+      no_ans_boost: -10
+      model_name_or_path: deepset/roberta-base-squad2
+      num_processes: 0
+  - name: ESRetriever
+    type: ElasticsearchRetriever
+    params:
+      document_store: DocumentStore
+      custom_query: null
+  - name: DocumentStore
+    type: ElasticsearchDocumentStore
+    params:
+      index: haystack_test
+      label_index: haystack_test_label
+  - name: PDFConverter
+    type: PDFToTextConverter
+    params:
+      remove_numeric_tables: false
+  - name: Preprocessor
+    type: PreProcessor
+    params:
+      clean_whitespace: true
+  - name: IndexTimeDocumentClassifier
+    type: TransformersDocumentClassifier
+    params:
+      batch_size: 16
+      use_gpu: -1
```

Comment:
shouldn't that be `false`?

Reply:
Good question :monocle_face: I don't know actually. I copied this file from the test resources... Going to investigate quickly